datasource-sdk/src/test/java/org/apache/kylin/sdk/datasource/framework/conv/PostgresqlSqlConverterTest.java

```diff
@@ -24,28 +24,41 @@
 import java.util.LinkedList;
 import java.util.List;
 
-public class GenericSqlConverterTest {
+public class PostgresqlSqlConverterTest {
```

Comment:
Is there a better way other than rename this testcase's name? Rename it is not necessary in my view.